arch/armv7m/jmp.S

```diff
@@ -14,29 +14,28 @@
  * %LICENSE%
  */
 
- /* setjmp env buffer description
-  * Offset  Description
-  * 0       Signal mask flag - if != 0 then next value stores a signal mask
-  * 4       Signal mask
-  * 8       SP register
-  * 12-43   r4-r11 registers
-  * 44      LR register
-  * 48-79   d8-d15 registers
-  * 80      fpscr register
-  */
+/* setjmp env buffer description
+ * Offset  Description
+ * 0       Signal mask flag - if != 0 then next value stores a signal mask
+ * 4       Signal mask
+ * 8       SP register
+ * 12-43   r4-r11 registers
+ * 44      LR register
+ * 48-103   d8-d15 registers
```

Comment:
As with armv7a, imho the previous one was correct. One space needs to be removed

Reply:
hmm, as per the [armv7-m manual](https://developer.arm.com/documentation/ddi0403/d/Application-Level-Architecture/Application-Level-Programmers--Model/The-optional-Floating-point-extension/The-FP-extension-registers?lang=en):
> Software can access the FP extension register bank as:
> - thirty-two 32-bit single-precision registers, S0-S31
> - sixteen 64-bit doubleword registers, D0-D15.

So these are 8-byte each

Reply:
Right, sorry